IPs/RSKIP_tbd.md

```diff
@@ -0,0 +1,282 @@
+As mentioned in the previous section, nowadays once the election takes place, the process is irreversible.
+This RSKIP proposes an automated and decentralized process to add one more layer of security to this process to avoid any human mistake putting funds at risk.
+
+This process protects funds from:
+- An invalid redeemscript that would render the funds burned.
+- A composition of pegnatories that are not responsive.
+
+This process doesn't include:
+- Verifying proposed new members use a valid HSM version.
+
+## Specification
+
+This RSKIP proposes changes in the Powpeg change process. The subsections will expand on the changes for each step.
+
+The election process in itself doesn't change. The authorized members will still propose a new composition, once the voting finishes the new composition will be committed and the Powpeg change will start.
+
+The current process consists in the following phases:
+1. Election start: m/n authorized members propose to start a new election.
+2. Composition proposal: m/n vote for the new composition.
+3. New composition commitment: m/n vote to commit new composition. New elections are disabled.
+4. Pre activation phase: immediately after commitment and during a number of blocks. No real change for end users.
+5. Activation phase: new Powpeg address activates, two-way peg starts operating from this new address.
+6. Migration phase: Funds from the retiring Powpeg wallet are moved to the active Powpeg.
+7. Post migration phase: Retiring Powpeg information is removed from the Bridge. New elections are re-enabled.
+
+### Commit new Powpeg
+
+The commitment of a new Powpeg composition remains almost unchanged except for a subtle need.
+Given the election could be reverted (see [here](#election-rollback)) the Bridge will have to store the last retired Powpeg script in a backup until the validation phase finishes.
+
+**Important**: This needs to take place before the current powpeg p2sh script is written in the storage.
+
+Suggested implementation:
+```
+KEY: "SVP_LAST_RETIRED_P2SHSCRIPT_BKP"
+VALUE: value of lastRetiredFedP2SHScript before commitment of new Powpeg
+```
+
+
+### New phase - Validation
+
+If this RSKIP is implemented a new phase will be added, in between the _pre activation_ and _activation_ phases, the _validation_ phase. This phase will, in a sense, replace the _preactivation_ phase.
+
+During a certain number of blocks a new protocol will be started, Spendability Validation Protocol (SVP).
+If this protocol succeeds, the Bridge will progress to the next phase (_activation_).
+If this protocol fails the election will be rollbacked.
+
+### Spendability Validation Protocol (SVP)
+
+The Spendability Validation Protocol is a new process in which the soon-to-be-retired Powpeg will send funds to the newly elected Powpeg, for it to send them back, proving that it can truly hold and spend funds, before proceeding with the activation and migration of the funds in the wallet.
```

Comment:
i will say "the current (soon-to-be-retired) Powpeg" since it is called "current powpeg" just after